Competition2017/src/competition/subsystems/shoot_fuel/ShootFuelForNSecondsCommandGroup.java

```diff
@@ -0,0 +1,21 @@
+package competition.subsystems.shoot_fuel;
+
+import com.google.inject.Inject;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class ShootFuelForNSecondsCommandGroup extends CommandGroup {
+    private DoubleProperty nSeconds;
+    
+    @Inject
+    public ShootFuelForNSecondsCommandGroup(XPropertyManager propManager, 
+            RightShootFuelCommandGroup rightShootFuelCommandGroup,
+            LeftShootFuelCommandGroup leftShootFuelCommandGroup) {
+        nSeconds = propManager.createPersistentProperty("the interval for shooting fuel in auto", 10);
```

Comment:
Make this less definite (linguistically speaking); something like "Autonomous shoot fuel duration" maybe.